src/main/java/uk/gov/hmcts/reform/sscs/mappers/ServiceHearingValuesMapper.java

```diff
@@ -205,119 +108,6 @@ private static String getListingComments(Appeal appeal, List<CcdValue<OtherParty
         return listingComments.isEmpty() ? null : String.join("\n", listingComments);
     }
 
```

Comment:
Might be worth using:
https://github.com/hmcts/sscs-hearings-api/blob/master/src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java#L176-L190

Reply:
That makes sense, will implement.